doc/users/next_whats_new/imshow_m_n_1.rst

```diff
@@ -0,0 +1,6 @@
+Imshow now coerces 3D arrays with depth 1 to 2D
+------------------------------------------------
+The number of ticks on colorbars was appropriate for a large colorbar, but
+Starting from this version arrays of size MxNx1 will be coerced into MxN 
+for displaying. This means commands like ``plt.imshow(np.ones((3, 3, 1)))`` 
```

Comment:
`np.ones((3, 3, 1))` may be bad example because colormapping will give unexpected results for identical values. Maybe `np.random.rand(3, 3, 1)` instead?